knowledge-base/chart-dash-series.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Chart: Dashed line series
+description: Modify the line chart to have dashed lines by using the native chart.
+type: how-to
+page_title: Dashed line chart
+slug: chart-dash-series
+position: 
+tags: MAUI
+ticketid: 1582451
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product Version</td>
+			<td>3.2.1</td>
+		</tr>
+		<tr>
+			<td>Product</td>
+			<td>Chart for .NET MAUI</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+
+In this article I will show you how to define a stroke for the Line Series Chart to change its appearance to that of a Dashed Chart.
+
+## Solution
+
+This solution uses the Native Chart Classes and their respective property/method for DashedArray to achieve the desired Dashed Line Series behavior.
+
+## Code
```

Comment:
we do not need Code to be heading 2
instead of ## 
use Here is the implementation you need to add in the project in order to achieve dashed series:
Also add the xaml used.